vmdb/db/migrate/20150414094834_create_container_projects.rb

```diff
@@ -0,0 +1,18 @@
+class CreateContainerProjects < ActiveRecord::Migration
+  def up
+    create_table :container_projects do |t|
```

Comment:
We have to keep in mind that the idea was to abstract these concepts, and not to create 1:1 tables with the current solution we're using (for example `container_group` is used for kubernetes pods and amazon tasks).

How data is displayed has nothing to do with how it's stored. If we decide that for clarity we need both "Projects" and "Namespaces" in the menu (I don't think we need both, just as we don't mention other entities as "Pods") then we can have them regardless of how many tables we use and how we deal with entities.

I am fine with maintaining this table and name `container_projects` and use it for namespaces as well.